CHANGES.md

```diff
@@ -1,4 +1,189 @@
+    resources lack a `Kind` property (thanks @Pothulapati!)
+  * The `inject` command proxy options are now converted into config
+    annotations; the annotations ensure that these configs are persisted in
+    subsequent resource updates
+  * Changed `inject` to require fetching a configuration from the control plane;
+    this can be disabled with the `--ignore-cluster` and `--disable-identity`
+    flags, though this will prevent the injected pods from participating in mesh
+    identity
+  * Included kubectl version check as part of `linkerd check` (thanks @yb172!)
+  * Updated `linkerd check` to ensure hint URLs are displayed for RPC checks
+  * Fixed sporadic (and harmless) race condition error in `linkerd check`
+  * Introduced a check for NET_ADMIN in `linkerd check`
+  * Fixed permissions check for CRDs
+  * Updated the `linkerd dashboard` command to serve the dashboard on a fixed
+    port, allowing it to leverage browser local storage for user settings
+  * Updated the `linkerd routes` command to display rows for routes that are not
+    receiving any traffic
+  * Added TCP stats to the stat command, under the `-o wide` and `-o json` flags
+  * The `stat` command now always shows the number of open TCP connections
+  * Removed TLS metrics from the `stat` command; this is in preparation for
+    surfacing identity metrics in a clearer way
+  * Unhid the `install-cni` command and its flags, and tweaked their descriptions
+  * Eliminated false-positive vulnerability warnings related to go.uuid
+* Controller
+  * Added a new public API endpoint for fetching control plane configuration
+  * **Breaking change** Removed support for running the control plane in
+    single-namespace mode, which was severely limited in the number of features
+    it supported due to not having access to cluster-wide resources
+  * Updated automatic proxy injection and CLI injection to support overriding
+    inject defaults via pod spec annotations
+  * Added support for the `config.linkerd.io/proxy-version` annotation on pod
+    specs; this will override the injected proxy version
+  * The auto-inject admission controller webhook is updated to watch pods
+    creation and update events; with this change, proxy auto-injection now works
+    for all kinds of workloads, including StatefulSets, DaemonSets, Jobs, etc
+  * Service profile validation is now performed via a webhook endpoint; this
+    prevents Kubernetes from accepting invalid service profiles
+  * Changed the default CPU request from `10m` to `100m` for HA deployments;
+    this will help some intermittent liveness/readiness probes from failing due
+    to tight resource constraints
+  * Updated destination service to return TLS identities only when the
+    destination pod is TLS-aware and is in the same controller namespace
+  * Lessen klog level to improve security
+  * Updated control-plane components to query Kubernetes at startup to determine
```

Comment:
Question, should there be a dash between `control` and `plane` here? (I know it was taken from a previous release)

Reply:
good point! i see 2 occurrences of `control-plane` and 56 of `control plane` so i think we know which one wins